coffee_github/src/repository.rs

```diff
@@ -116,22 +117,53 @@ impl Github {
                         let files = WalkDir::new(plugin_path.path()).max_depth(1);
                         for file in files {
                             let file_dir = file.unwrap().clone();
-                            let (tmp_path_to_plugin, tmp_plugin_name) =
+                            let (tmp_root_path, tmp_plugin_name) =
                                 get_plugin_info_from_path(file_dir.path()).unwrap();
-                            path_to_plugin = Some(tmp_path_to_plugin.to_string());
                             plugin_name = Some(tmp_plugin_name.to_string());
-                            debug!("looking for {tmp_plugin_name} in {tmp_path_to_plugin}");
+                            debug!("looking for {tmp_plugin_name} in {tmp_root_path}");
                             let file_name = file_dir.file_name().to_str().unwrap();
-                            plugin_lang = match file_name {
-                                "requirements.txt" => PluginLang::PyPip,
-                                "pyproject.toml" => PluginLang::PyPoetry,
-                                "go.mod" => PluginLang::Go,
-                                "cargo.toml" => PluginLang::Rust,
-                                "pubspec.yaml" => PluginLang::Dart,
-                                "package.json" => PluginLang::JavaScript,
-                                "tsconfig.json" => PluginLang::TypeScript,
-                                _ => PluginLang::Unknown,
-                            };
+                            match file_name {
+                                "requirements.txt" => {
+                                    plugin_lang = PluginLang::PyPip;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.py"))
+                                }
+                                "pyproject.toml" => {
+                                    plugin_lang = PluginLang::PyPoetry;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.py"))
+                                }
+                                "go.mod" => {
+                                    plugin_lang = PluginLang::Go;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.go"))
```

Comment:
```suggestion
                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.go"))
```

wait what happens here? `.go`? go use the binary this is wrong